.github/workflows/backend-tests.yml

```
@@ -19,21 +19,21 @@ jobs:
         uses: erlef/setup-beam@v1
         with:
           otp-version: "25.0"
-          elixir-version: "1.15.7"
+          elixir-version: "1.19.5"
 
       - name: Cache deps
         uses: actions/cache@v3
```

actionlint (1.7.11)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)